src/modules/ProductDetails/VariantsSection/VariantsSection.module.scss

```diff
@@ -46,49 +54,170 @@
     .squareImageWrapper {
       position: relative;
       flex-shrink: 0;
-      
+
       width: 100%;
       height: 0;
       padding-bottom: 100%;
       overflow: hidden;
 
       .squareImage {
         position: absolute;
-        
+
+        object-fit: contain;
         border: var(--elements) solid 1px;
         padding: 7%;
         width: 100%;
         height: 100%;
-        object-fit: contain;
       }
-      
+
       .selectedSquareImage {
         border: black solid 1px;
       }
     }
   }
-  
+
   .selectedImageWrapper {
     display: flex;
     justify-content: center;
+
     height: 100%;
     margin: 0 auto;
-    
+
     .selectedImage {
+      object-fit: contain;
       width: 100%;
       height: 100%;
-      object-fit: contain;
       max-height: 465px;
-    
-      @include tablet {
-        max-height: 350px;
+
+      @include mobile {
+        max-height: 365px;
       }
+    }
+  }
+}
+
+.availableVariantsWrapper {
+  display: flex;
+  flex-direction: column;
+
+  max-width: 512px;
+  width: 100%;
+  color: var(--secondary);
+
+  @include mobile {
+    max-width: 100%;
+  }
     
-      @include mobile-small {
-        max-height: 290px;
-      }
+  .categoryText {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 8px;
+
+    @include mobile {
+      max-width: 560px;
+    }
+  }
+
+  .colorRadioWrapper {
+    position: relative;
+    display: flex;
+    gap: 8px;
+
+    margin-bottom: 48px;
+  }
+
+  .capacityContainer {
+    position: relative;
+    max-width: 320px;
+    margin-bottom: 32px;
+
+    @include mobile {
+      max-width: 560px;
+    }
+
+    &::before {
```

Comment:
Write values that you have the same in before and after together. 

&:: before,
&::after {
content: “”;
}

Reply:
But in this case we have the same position of before and after, so only one gray line on layout

![Uploading Screenshot 2024-04-22 at 15.05.55.png…]()